src/components/button/button-styles.jsx

```diff
@@ -23,6 +23,9 @@ const buttonModifierFn = (variant, color, colorFocus, colorDisabled, textColorDi
 
       '&:focus, &:active': {
         boxShadow: `0 0 0 2px ${colorFocus}`,
+        '& svg': {
```

Comment:
What do you think about "protecting" these `transform`s so that we only use them in `iconButton`? These will affect all svgs inside the button atm (say you put an svg inside a normal button without `iconButton` being set)

Reply:
I find the SVG zooming behavior to be weird in the first place. We already have the outer border change so the active state is apparent without the scaling of the svg. Which designer would be the person to talk about the desired behavior of these buttons?

If that person says that we should keep it then we could implement them so that they only impact `iconButton` buttons.

Reply:
Fully agree, lets find someone and sync!